internal/commands/create.go

```diff
@@ -224,6 +226,62 @@ func getRegoFilePaths(path string) ([]string, error) {
 	return regoFilePaths, nil
 }
 
+func parsePolicies(policyPaths []string, libraryPaths []string) ([]*regoPolicy, error) {
+	policies, err := loadPolicies(policyPaths)
+	if err != nil {
+		return nil, err
+	}
+	libraries, err := loadPolicies(libraryPaths)
+	if err != nil {
+		return nil, err
+	}
+
+	for _, p := range policies {
+		if len(p.policy.Imports) > 0 {
+			for _, i := range p.policy.Imports {
+				library := getLibrary(libraries, i.Path.String())
+				if library == nil {
+					return nil, fmt.Errorf("imported library %s not found", i.Path.String())
+				}
+
+				// We read the file again from disk to perserve the formatting of the policy
+				// The OPA parser removes a lot of the nice syntax sugar that makes it easier for us to read
+				// ---
+				// We just read the library a few milliseconds ago, assuming errors won't happen on the second read
+				data, _ := ioutil.ReadFile(library.path)
+				p.libraries = append(p.libraries, string(data))
+			}
+		}
+	}
+
+	return policies, nil
+}
+
+func loadPolicies(paths []string) ([]*regoPolicy, error) {
+	var policies []*regoPolicy
+	for _, file := range paths {
+		data, err := ioutil.ReadFile(file)
+		if err != nil {
+			return nil, err
+		}
+		policy, err := ast.ParseModule("", string(data))
```

Comment:
nit: I think we could get away with just using `ast.ParseImports` here. I'm not seeing us referencing anything else, so it would show a bit more intent and side step the whole passing a blank file

Reply:
At one point, I was using the `*ast.Module.String()` method to avoid the duplicate reads from disk, but since it changes the formatting I opted not to and never changed this. I'll update this.

Reply:
Actually, we use the package path in the `getLibrary()` lookup, which isn't possible if we only process the imports, so I think we should leave this as it is.